tv_apps/app-deck/js/app_deck.js

```diff
@@ -206,12 +208,25 @@
     },
 
     onAppUninstalled: function ad_onAppUninstalled(apps) {
-      // TODO: uninstall an app will be fixed in bug 1115633
+      var that = this;
+      var appGridElements = apps.map(this._findAppGridElement.bind(this));
+      appGridElements.forEach(function(elem) {
+        if (elem.dataset.revokableURL) {
+          URL.revokeObjectURL(elem.dataset.revokableURL);
+        }
+        // Move focus to next or previous element of `elem`, because
+        // we are going to remove `elem` from DOM tree
+        var nextFocus = elem.nextSibling || elem.previousSibling;
+        that._spatialNavigator.focus(nextFocus);
+
+        that._spatialNavigator.remove(elem);
+        that._appDeckGridViewElem.removeChild(elem);
+      });
     },
 
     _findAllAppGridElements: function ad_findallAppGridElements() {
       return SharedUtils.nodeListToArray(
-        document.getElementsByTagName('app-button'));
+        document.querySelectorAll('smart-button.app-button'));
```

Comment:
`quesrSelector` doesn't performs as good as `getElementsByTagName`. Does it work with `this. _appDeckGridViewElem.getElementsByTagName('smart-button')` ?
This is not critical so it's just a suggestion.